core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -174,14 +174,21 @@ org.jenkinsci.jruby.JRubyMapper$DynamicProxy
 org.jenkinsci.lib.xtrigger.XTriggerCause
 org.jenkinsci.lib.xtrigger.XTriggerCauseAction
 
-# TODO remove (also XStream2.BlacklistedTypesConverter.JRUBY_PROXY) when https://github.com/jenkinsci/ruby-runtime-plugin/pull/5 is widely adopted
+# TODO remove (also XStream2.BlacklistedTypesConverter.JRUBY_PROXY) when Ruby Runtime is fixed
+# Related PRs:
+#   - https://github.com/jenkinsci/ruby-runtime-plugin/pull/5,
+#   - https://github.com/jenkinsci/ruby-runtime-plugin/pull/6
+#
+# oleg:nenashev in PR#6 we are trying to get help from last maintainers due to the plugin codebase splitbrain.
```

Comment:
`oleg-nenashev`?

Reply:
I have different symbols in GitHub and LDAP accounts, so whatever works :D Will fix it